js/components/pages/PassagePage.react.js

```diff
@@ -112,6 +112,8 @@ export class PassagePage extends Component {
           </div>
 
           <div className="meta-information">
+            <a className="left" href="http://www.esv.org">ESV</a>
+            <a className="right" href="#">Help</a>
```

Comment:
That seems reasonable, I'll plan on getting the instruction page working tomorrow morning.